pkg/core/contract_listener.go

```diff
@@ -44,7 +44,8 @@ type ContractListenerWithStatus struct {
 	Status interface{} `ffstruct:"ContractListenerWithStatus" json:"status,omitempty" ffexcludeinput:"true"`
 }
 type ContractListenerOptions struct {
-	FirstEvent string `ffstruct:"ContractListenerOptions" json:"firstEvent,omitempty"`
+	FirstEvent          string           `ffstruct:"ContractListenerOptions" json:"firstEvent,omitempty"`
+	SubscriptionOptions *fftypes.JSONAny `ffstruct:"ContractListenerOptions" json:"subscriptionOptions,omitempty"`
```

Comment:
I'm struggling with what this field should be named. For context, in the older ETHConnect codebase we used to call these "subscriptions". In the newer FFTM API (which EVMConnect is based on), we now refer to these as "listeners" to help distinguish them from FireFly Subscriptions (which are a higher level object). So this field actually ends up maping to `Listener.Options`
https://github.com/hyperledger/firefly-transaction-manager/blob/fe5ace887c9f541c3053fd31d937d46ce6adc3c9/pkg/apitypes/api_types.go#L212

But calling this field `ContractListenerOptions.ListenerOptions` seems a bit redundant 🤔 

@awrichar or @peterbroadhurst Do you have any suggestions here?

Reply:
@nguyer - sorry I missed this before.

I think the consistent pattern here with other things, would be that the `options` are passed through to EVMConnect. If you look at `invoke` APIs, or the tokens - that's what we do I believe.